que.go

```diff
@@ -155,44 +155,31 @@ func execEnqueue(j *Job, q queryable) error {
 		return ErrMissingType
 	}
 
-	queue := pgx.NullString{
+	queue := sql.NullString{
 		String: j.Queue,
 		Valid:  j.Queue != "",
 	}
-	priority := pgx.NullInt16{
-		Int16: int16(j.Priority),
+	priority := sql.NullInt64{
+		Int64: int64(j.Priority),
 		Valid: j.Priority != 0,
 	}
-	runAt := pgx.NullTime{
-		Time:  j.RunAt,
-		Valid: !j.RunAt.IsZero(),
+	runAt := sql.NullString{
+		String: j.RunAt.Format(time.RFC3339),
```

Comment:
@cbandy yes I'll change it. If `time.RFC3339Nano` is used here, I guess there is no need of truncating it to second in the tests.